p2p_wallet/Scenes/Main/Send/Input/SendInputCoordinator.swift

```diff
@@ -102,7 +103,7 @@ final class SendInputCoordinator: Coordinator<SendResult> {
             Task { await viewModel.load() }
         }
 
-        return subject.prefix(1).eraseToAnyPublisher()
+        return subject.eraseToAnyPublisher()
```

Comment:
This breaks current logics for `normal send`, as subject.send from other place won't trigger completion, need to keep this SendInputCoordinator untouched